src/popup.html

```diff
@@ -34,10 +39,22 @@
             {{/buckets}}
           </li>
         {{/abTests}}
+
         </ul>
       </div>
     {{/abTests.length}}
-    <ul class='external-links'>{{#externalLinks}}<li><a href="{{url}}" class="external">{{{name}}}</a></li>{{/externalLinks}}</ul>
+
+    <ul class='add-top-border'>
+      {{#externalLinks}}
+        <li><a href="{{url}}" class="external">{{{name}}}</a></li>
+      {{/externalLinks}}
+    </ul>
+
+    <ul class='add-top-border'>
+      <li>
+        <div id='highlight-components'></div>
```

Comment:
Why not make this a standard link with link text?

Reply:
Because if you use a link and click it, it will redirect the page. It is empty because the state of the toggle lives on the webpage itself and needs to be queried using message passing.

Reply:
You can use JS to catch the click event, prevent the default action and do something different.